CMakeLists.txt

```diff
@@ -1939,59 +1094,6 @@ else()
     # #1286: invalid attribute
     hpx_add_compile_flag_if_available(-wd1286)
   endif()
-
-  set(__has_timestamp_support OFF)
-  include(TargetArch)
-
-  target_architecture(__target_arch)
-  if("${__target_arch}" STREQUAL "i386"
```

Comment:
4 occurrences of `__target_arch` are still here in the `CMakeLists.txt`, I'm not sure it was intended